Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -0,0 +1,319 @@
+import base64
+from datetime import datetime
+import dateutil.parser
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+from typing import Any, Dict, Union, Optional, Tuple, List
+
+FILTER_RELATIONSHIP_AND = "FILTER_RELATIONSHIP_AND"
+
+DEMISTO_OCCURRED_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+RECO_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DEMISTO_INFORMATIONAL = 0.5
+RECO_API_TIMEOUT = 30  # Increase timeout for RECO API
```

Comment:
When defining a time const, either use a native time type (preferable) or rename the var to include the time unit (`RECO_API_TIMEOUT_IN_SECONDS`)